composer.json

```diff
@@ -37,7 +37,7 @@
       "Dealerdirect\\Composer\\Plugin\\Installers\\PHPCodeSniffer\\Plugin::run"
     ],
     "prefix-mf2": [
-			"@php ./vendor/humbug/php-scoper/bin/php-scoper add-prefix --prefix=Webmention --output-dir=./libraries/mf2 --config=config/php-scoper/mf2.inc.php --force --quiet"
+			"@php ./vendor/humbug/php-scoper/bin/php-scoper add-prefix --prefix=Webmention --output-dir=./includes/handlers/libraries/mf2 --config=config/php-scoper/mf2.inc.php --force --quiet"
```

Comment:
I would not move it... Yes it is used by a handler, but that does not mean that it has to be in a sub-folder of the handlers... I would keep it in libraries (root folder)...